#ifndef DIALOG_H
#define DIALOG_H

#include <QDialog>
#include<QtGui>
#include<QtCore>

namespace Ui {
class Dialog;
}

class Dialog : public QDialog
{
    Q_OBJECT
    
public:
    explicit Dialog(QWidget *parent = 0);
    ~Dialog();

    void addRoot(QString name, QString description);
    void addChild(QTreeWidgetItem *parent,QString name, QString description);
    

private slots:
    void on_pushButton_clicked();

private:
    Ui::Dialog *ui;
};

#endif // DIALOG_H
